utils/docker.go

```diff
@@ -291,9 +291,13 @@ func StopContainer(container types.Container) {
 		errors.CheckErr(err, 108, "")
 	}
 
-	// Remove the container so it isnt lingering in the background
-	if err := cli.ContainerRemove(ctx, container.ID, types.ContainerRemoveOptions{}); err != nil {
-		errors.CheckErr(err, 108, "")
+	// Do not attempt to remove appsody images as that happens automatically
+	// when an appsody container stops
+	if !strings.HasPrefix(container.Image, "appsody") {
```

Comment:
Rather than hardcode appsody here, could we allow the operation to succeed even if it fails to remove container?  (There could be other future extensions that starts a container with `--rm` flag which self-removes a stopped container)

Reply:
we could but I would rather add that as a new enhancement rather than keep this one going